.github/workflows/coprocessor-benchmark-gpu.yml

```diff
@@ -0,0 +1,279 @@
+# Run all fhevm coprocessor benchmarks on a GPU instance on Hyperstack and return parsed results to Slab CI bot.
+name: coprocessor-benchmark-gpu
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    # Nightly tests @ 1AM after each work day
+    - cron: "0 1 * * MON-FRI"
```

Comment:
These are going to run on SXM5 VMs, it might become expensive: maybe it'd be better to have them run weekly instead of nightly :thinking:

Reply:
I changed to weekly